executor/builder.go

```diff
@@ -1184,6 +1201,7 @@ func (b *executorBuilder) buildExists(v *plan.PhysicalExists) Executor {
 	e := &ExistsExec{
 		baseExecutor: newBaseExecutor(b.ctx, v.Schema(), v.ExplainID(), childExec),
 	}
+	e.chunkCap = 1
```

Comment:
If just assigning a const to a variable of a struct, I feel like it is better to put it as part of the creation of such struct.